lib/checkr.rb

```diff
@@ -16,6 +16,7 @@
 require 'checkr/api_singleton'
 require 'checkr/api_list'
 require 'checkr/util'
+require 'pry'
```

Comment:
why do we need to include `pry` in production code?

Reply:
@joecabezas Just removed it, forgot to do it when i first pushed.